src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  onTodoReset: (value: null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onTodoReset }) => {
+  const [userFromServer, setUserFromServer] = useState<User | null>(null);
+  const [userLoading, setUserLoading] = useState(true);
```

Comment:
```suggestion
  const [isUserLoading, setIsUserLoading] = useState(true);
```
Boolean variables should start from `is` or `has`